Document how to see available namespaces and set a default (#86)
alphagov/govuk-kubernetes-cluster-user-docs: source/manage-app/get-app-info/index.html.md

Most of the following examples use the Kubernetes command-line tool, `kubectl`.

## Choose and set a namespace

> To see a list of all namespaces, run the following in the command line:
> 
> `kubectl get namespaces`
> 

Reviewer: I think it's better to omit this part, since anyone reading these docs should never need to touch anything other than the `apps` namespace and that's the only one you'd want to set as your default.

```suggestion
```

Author: I've moved this further down as an incidental note, just if people are curious (as I was, and I suspect others will be). What do you think?

Reviewer: I think it's probably best to resist the temptation to include extra detail for curiosity/interest, as that bulks out the documentation and adds up to a document that's actually harder for people to use.

Also as a rule of thumb, it's a good idea to avoid documenting anything that's non-essential and discoverable in just a few seconds via a [web search](https://www.google.com/search?q=kubernetes+list+namespaces).

Author: Removed, and just noticed you'd already approved, so apologies for requesting the re-review :) merging

> The most common namespace you'll use is the `apps` namespace, which you'll specify as `-n apps` in the commands further on in this document. You can omit that flag by setting `apps` as your default namespace:

Reviewer: ```suggestion
All the GOV.UK applications are in the `apps` namespace. You can avoid the need to specify `-n apps` in every `kubectl` command by setting `apps` as your default namespace:
```

`kubectl config set-context --current --namespace=apps`

## Get a list of all deployed apps in a namespace

> To get a list of all deployed apps in a namespace, run:
> 
> ```sh
> kubectl -n <namespace> get deployments

Reviewer: ```suggestion
To get a list of all deployed apps, run:

```sh
kubectl -n apps get deployments
```
